OWNERS_ALIASES

```diff
@@ -1,18 +1,11 @@
 # See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#OWNERS_ALIASES
 
 aliases:
+  cincinnati-graph-data-reviewers:
   cincinnati-graph-data-approvers:
-    - crawford
```

Comment:
@crawford Don't know if you wanted to be left in review/approval for graph changes or not. PTAL